.azure/pipelines/quarantined-tests.yml

```diff
@@ -11,11 +11,20 @@ pr:
     include:
     - '*'
 
+schedules:
+- cron: "0 */4 * * *"
+  displayName: Every 4 hours test run
+  branches:
+    include:
+    - master
```

Comment:
@Pilchie @mkArtakMSFT do we have enough Blazor tests running in Helix to make including schedule runs for the 'blazor-wasm' branch worthwhile?

Reply:
I don't think we have any running https://github.com/dotnet/aspnetcore/pull/18636

Reply:
@BrennanConroy you're probably right but I wasn't positive on the current status